### ISSUE TITLE
dorado.0.4.2 segmenetation fault with stereo duplex

### ISSUE THREAD

**Issue by juzuo-git** [nanoporetech/dorado#464]
Hi,
When I running `dorado duplex ` I got error : segmentation fault (core dumped). Could you help me to solve this problem?
My data was produced by miniON in `5khz`, and kit is `SQK-NBD114-96`

I had check the issues list, and my issue is different with #162 , which was caused by 4/5khz 
**This is the code:**
```shell
/Program/dorado-0.4.2-linux-x64/bin/dorado duplex \
/Program/dorado-0.4.2-linux-x64/Data/dna_r10.4.1_e8.2_400bps_sup@v4.2.0 \
 fast5/ \
> duplex_result.bam
``` 

**This is the full error message:**
<details><summary>Details</summary>
<p>
[2023-11-09 18:18:00.679] [info] > No duplex pairs file provided, pairing will be performed automatically
[2023-11-09 18:18:31.702] [info]  - set batch size for cuda:0 to 640
[2023-11-09 18:18:37.516] [info]  - set batch size for cuda:0 to 1664
[2023-11-09 18:18:37.516] [info] > Starting Stereo Duplex pipeline
[2023-11-09 18:18:37.542] [info] > Reading read channel info
[2023-11-09 18:18:37.542] [info] > Processed read channel info
/scem/work/.lsbatch/1699525074.452260.shell: line 10: 236377 Segmentation fault      (core dumped) /Program/dorado-0.4.2-linux-x64/bin/dorado duplex /Program/dorado-0.4.2-linux-x64/Data/dna_r10.4.1_e8.2_400bps_sup@v4.2.0 fast5/ > duplex_res.bam
</p>
</details> 

But the scripts below about basecalling (normal basecalling and modifeid calling) were **running successfully**:
```shell
/Program/dorado-0.4.2-linux-x64/bin/dorado basecaller \
--barcode-both-ends \
  --kit-name SQK-NBD114-96 \
  /Program/dorado-0.4.2-linux-x64/Data/dna_r10.4.1_e8.2_400bps_sup@v4.2.0 \
  fast5/ \
  > kit_res.bam
``` 
```shell
/Program/dorado-0.4.2-linux-x64/bin/dorado basecaller \
/Program/dorado-0.4.2-linux-x64/Data/dna_r10.4.1_e8.2_400bps_sup@v4.2.0 \
fast5 \
--modified-bases 6mA \
--reference /reference.fasta \
> test_6mA.bam
``` 

Any idea?
Looking forward to your replay.

**Comment by tijyojwad** [comment]
Hi @juzuo-git - I think the issue might be that you're using `fast5`. For `duplex` we need to load data in a particular order, and we skip any data that's in `fast5` format. If all your data is in fast5, then we skip everything and that's probably what's causing the segfault. Can you convert your data to pod5 and try again?

We will add better error handling for this case so it's clearer what needs to be done.

**Comment by tijyojwad** [comment]
Hi @juzuo-git - the segfault is now fixed in dorado [v0.4.3](https://github.com/nanoporetech/dorado#installation).

**Comment by juzuo-git** [comment]
Thx! After I transfer fast5 to pod5, I run code successed！

**Comment by juzuo-git** [comment]
> Hi @juzuo-git - the segfault is now fixed in dorado [v0.4.3](https://github.com/nanoporetech/dorado#installation).

Hi, @tijyojwad - That's very great, I would like to try it latter.
